pppoe/pppoe.go

```diff
@@ -37,17 +38,28 @@ type Conn struct {
 	// PPPoE discovery protocol. We use this to set up a session, and
 	// to tear it down when we close the Conn.
 	discovery net.PacketConn
-	// addr is the address of the remote PPPoE concentrator. We use it
-	// during session teardown, but mostly it's there to provide if
-	// someone asks for Conn.RemoteAddr().
-	addr Addr
+	// localAddr is the address of the local PPPoE endpoint. It's here
+	// to provide in response to LocalAddr().
```

Comment:
Go style is to omit the parens when talking about the func/method itself, and only include them when you're talking about the result of the call.

But this isn't exported godoc so I'm not super fussed about it. And it's not in Go's tree either. :)

Reply:
(Also in commit message)